[()]
[()]
[(Error (load-ascii GroundingSpace-0x55e608a2e8b8 sample_kb_v2.metta) Exception caught:
RuntimeError: load_ascii: file not found

At:
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/python/hyperon/stdlib.py(167): load_ascii_atom
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/python/hyperon/atoms.py(469): execute
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/python/hyperon/atoms.py(234): _priv_call_execute_on_grounded_atom
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/python/hyperon/runner.py(209): run
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/python/hyperon/metta.py(35): main
  /mnt/c/Users/me/Documents/Prolog/metta/metta-wam-win/venv/bin/metta(8): <module>
)]
